install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -193,7 +181,14 @@ Gluster server administrators using normal POSIX compliant security.
 By default, SELinux does not allow writing from a pod to a remote Gluster
 server.
 
-To enable writing to GlusterFS volumes with SELinux enforcing on each node, run:
+To enable writing to GlusterFS volumes with SELinux enforcing first install
+the`docker-selinux on each node:
+
+----
+# yum install docker-selinux
```

Comment:
> Do we need to call it out separately here?

@erinboyd got into a situation where this package was not installed. Perhaps it is install on a new install but not an update. To be safe I think we should include it in the docs @adellape WDYT ?